crates/core/tedge/Cargo.toml

```diff
@@ -14,7 +14,6 @@ readme = "README.md"
 anstyle = { workspace = true }
 anyhow = { workspace = true }
 base64 = { workspace = true }
-c8y-configuration-plugin = { workspace = true }
```

Comment:
The `packages.json` file still has references to the old plugin. If we remove this from here, they won't be built anymore, right?

Reply:
We still need to keep the older c8y-configuration-plugin in place for now as some people might be relying on this old interface to support custom child device implementations (as we [publicly documented the interface](https://thin-edge.github.io/thin-edge.io/references/c8y-configuration-management/))

Reply:
> We still need to keep the older c8y-configuration-plugin in place for now as some people might be relying on this old interface

Yeah, that's what I was expecting as well and got confused by this change, essentially skipping the build of this plugin, which we still want to package for those "special usecases". But then had another look at the `package_list.sh` changes, where this deprecated package is added to the consolidated package list that is finally built. So, it makes sense now.